kiwixbuild/versions.py

```diff
@@ -31,7 +31,7 @@
 # - set KIWIX_DESKTOP_RELEASE to 0
 
 release_versions = {
-    'libzim': None, # Depends of base deps (was 0)
+    'libzim': 0, # Depends of base deps (was None)
```

Comment:
I prefer to change this dictionary when we create a new release.

Reply:
Yep, just wanted to mention that it was required to build it.